### DISCUSSION TITLE
Sorry for the radio silence!

### DISCUSSION THREAD

**Discussion by eteran** [eteran/nedit-ng#307] [General]
Hey guys, glad to see that we haven't had too many new issues this past month. Sorry that I've been a bit missing, work and life got crazy busy, but I plan to get back into rapid development now that things have started to calm down :-).

**Comment by grege2** [comment]
Hey Evan, no problem. I'm happily using NG every day, loving it. (On Mac, Big Sur 11.2.1). So smooth. I'm enjoying re-learning the macro language and making a few convenience tools. I don't think I have any significant issues. Cheers.

**Comment by 1div0** [comment]
Hi++ @eteran !

I would like to help with l10n, m13n, i18n and Qt6.

Cheers.

**Comment by eteran** [comment]
@1div0 it's been a million years, but and Qt6 is now supported, but if you are interested in translations, help is always welcome!

**Reply by 1div0** [reply]
100 years approximately. In the binary. You know, there are only 10 types of people.
